app/js/controllers/relatedCtrl.js

```diff
@@ -3,10 +3,11 @@ KMCMenu.controller('relatedCtrl', ['$scope',
 		$scope.relatedOption = "relatedToEntry";
 		$scope.entryList = "";
 		$scope.playlistId = {id:'' ,text:''};
+		$scope.playerData.config.plugins["related"] = {disable: true};
 
 		// init radio buttons according to selected related entries source
-		if ($scope.playerData.config.player.plugins && $scope.playerData.config.player.plugins["related"]) {
-			var data = $scope.playerData.config.player.plugins["related"];
+		if ($scope.playerData.config.plugins && $scope.playerData.config.plugins["related"]) {
+			var data = $scope.playerData.config.plugins["related"];
```

Comment:
why var ?

Reply:
i focused on the logic and not to fix the entire studio

Reply:
the all files in the repo are the same using var, it is old code and angular 1, we can open task of refactor